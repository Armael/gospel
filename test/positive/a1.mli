type t

(* EXPECTED
   [0] OK
*)
