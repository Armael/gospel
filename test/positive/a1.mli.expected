
*******************************
********** Parsed file ********
*******************************
type t

*******************************
****** GOSPEL translation *****
*******************************
(*@ open Stdlib *)

(*@ open Gospelstdlib *)

type t
  

*******************************
********* Typed GOSPEL ********
*******************************
module a1.mli

  Namespace: a1.mli
    Type symbols
       t
      
    Logic Symbols
      
    Field Symbols
      
    Exception Symbols
      
    Namespaces
      
    Type Namespaces
      
  Signatures
    (*@ open Stdlib *)
    
    (*@ open Gospelstdlib *)
    
    type t
         

OK
